### DISCUSSION TITLE
Create a Github action to Test the framework

### DISCUSSION THREAD

**Discussion by IntegerAlex** [IntegerAlex/hasty-server#19] [Q&A]
Create a CI/action to test code maybe add jest or any other testing lib.
You can raw dogg test using Just Javascript.

**Comment by georgioupanayiotis** [comment]
@IntegerAlex can you assign me this ticket ?

**Comment by IntegerAlex** [comment]
Assigned.

**Reply by IntegerAlex** [reply]
dont make big commits

**Reply by IntegerAlex** [reply]
Make a structure and create a pR

**Comment by IntegerAlex** [comment]
any update

**Reply by georgioupanayiotis** [reply]
yes i will create a PR soon

**Comment by georgioupanayiotis** [comment]
created a PR here https://github.com/IntegerAlex/hasty-server/pull/24

**Comment by IntegerAlex** [comment]
But there are no test files, we need a full framework like which deps we will be using for testing and all.

**Comment by IntegerAlex** [comment]
I am not saying to write test, That i will do but create a structure you CI just spins a node:ubuntu and run test, It's valid but we don't have npm run test yet.

**Reply by IntegerAlex** [reply]
@georgioupanayiotis will you be able to make specified changes

**Comment by c4ist** [comment]
what do you need help with? @IntegerAlex

**Reply by IntegerAlex** [reply]
Create unit test with jest
